src/storages/redis-storage.ts

```diff
@@ -0,0 +1,63 @@
+import { StorageType } from "../constants/storage-type";
+import BaseStorage from "../storages/base-storage";
+
+class RedisStorage extends BaseStorage {
+  private redisClient: any;
```

Comment:
```suggestion
  private redisClient: any;
```
Can we remove any here ?